Src/AutoFixture/IFixtureExtensions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Ploeh.AutoFixture
+{
+    public static class IFixtureExtensions
```

Comment:
Inspired by Haskell's [replicate](http://zvon.org/other/haskell/Outputprelude/replicate_f.html), how about naming this `FixtureReplicator`, `Replicator`, or similar? Or maybe we could name the class `FixtureRepeater`, `Repeater`, and so on.

FWIW, we have another class with extension methods, and it's called [`FixtureFreezer`](https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture/FixtureFreezer.cs) instead of the general purpose XyzExtensions name.